tabs/auxiliary.js

```diff
@@ -49,19 +50,25 @@ TABS.auxiliary.initialize = function (callback) {
     }
     
     function configureRangeTemplate(auxChannelCount) {
-
         var rangeTemplate = $('#tab-auxiliary-templates .range');
         
         var channelList = $(rangeTemplate).find('.channel');
         var channelOptionTemplate = $(channelList).find('option');
         channelOptionTemplate.remove();
+
+        //add value to autodetect channel
+        var channelOption = channelOptionTemplate.clone();
+        channelOption.text('AUTO SELECT');
```

Comment:
The text `AUTO SELECT` is a bit too wide for the current drop down:

![image](https://user-images.githubusercontent.com/4742747/31866220-dad5b41c-b7d8-11e7-9f70-898337a574bd.png)

As a workaround, changing it to `AUTO` will work.

Also, please use `chrome.i18n.getMessage('versionLabelFirmware')` to load messages from the localised messages file.